README.md

```diff
@@ -316,6 +316,31 @@ monday.execute("notice", {
 });
 ```
 
+#### Open files preview dialog
+Opens a popup card with the preview of an asset
+
+**type**
+`'openFilesDialog'`
+
+**params**
+
+| Parameter|Type | Description | Required | Default Value |
+| --- | --- | --- | --- | --- |
+| boardId| Integer | The ID of the board | Yes | |
+| itemId| Integer | The ID of the item, which contains an asset | Yes | |
```

Comment:
item id and column id can be mandatory (see files-preview-logic-connector.js) -  if you find it hard for them to pass it